client/src/views/Hustenview/index.jsx

```diff
@@ -0,0 +1,68 @@
+import { hot } from 'react-hot-loader';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+import { Header } from '../../components';
+import Hint from '../../components/Hint';
+import classify from './classif.js';
+
+const Title = () => (
+  <Typography component="p" variant="subtitle1" color="inherit">
+    Hast Du Husten?
+  </Typography>
+);
+
+const MyRadio = ({ options, id, question, preselected, info }) => {
+  const renderedoptions = options.map((opt, idx) => <FormControlLabel value={idx} control={<Radio />} label={opt} />);
+  const [value, setValue] = React.useState(preselected);
+  const handleChange = (event) => {
+    setValue(parseInt(event.target.value));
+  };
+  return (
+    <FormControl component="fieldset">
+      <FormLabel component="legend">{question}</FormLabel>
+      {info != undefined ? <Hint text={info} /> : <span />}
+      <RadioGroup name={id} value={value} onChange={handleChange}>
+        {renderedoptions}
+      </RadioGroup>
+    </FormControl>
+  );
+};
```

Comment:
For the code understanding it would be better if this is outsourced into a separate component. But nice to have

Reply:
The page is not large and remains quite readable (to me at least), I think separating a component is only meaningful when reuse outside that file is intended.